plugin/dns64/README.md

```diff
@@ -70,6 +72,16 @@ Enable translation even if an existing AAAA record is present.
 }
 ~~~
 
+Apply translation even to the requests which arrived over IPv4 network. Beware that enabling this option in CoreDNS handling combination 
+of dual-stack and IPv6 clients might cause problems by also doing DNS64 translations for requests of dual-stack clients.
```

Comment:
... but the plugin will do translations for dual-stack clients even if the option is not enabled. So, the behavior described is not specific to this option.  As long as a dual-stack client makes a query for a `AAAA` address over IPv6 transport, this plugin will attempt a translation.

The difference In behavior would be for `AAAA` queries made over IPv4 transport.  By default, the plugin gives a different answer to the client depending on the transport version. On the surface that seems potentially problematic - different answers for same name/type depending on which interface you send the request out on.

The `allow_ipv4` option makes the answer consistent for both transports.

Reply:
How about this wording:

```
Warning, the `allow_ipv4` feature will apply translations to requests coming from dual-stack clients. This means that a request for a client that sends an `AAAA` that would normal result in an `NXDOMAIN` would get a translated result. This may cause unwanted IPv6 dns64 traffic when a dualstack client would normally use the result of an `A` record request.
```

@chrisohaver Does this better explain it/warn the user?

Reply:
fwiw, I don’t understand. But if you are ok with it then let’s go with that.

Reply:
It's a warning because users with functioning IPv4 will get re-routed to IPv6 unnecessarily. That's why we implemented it the current way by default. This plugin is meant for users who are IPv6-only, not dualstack. If the user can reach IPv4, they should get untranslated results.

Reply:
I have adjusted wording as you suggested @SuperQ, please take a look if it makes sense